packages/core/contracts/oracle/implementation/DesignatedVotingV2.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../../common/implementation/MultiRole.sol";
+import "../../common/implementation/Stakeable.sol";
+import "../interfaces/VotingAncillaryInterface.sol";
+import "../interfaces/FinderInterface.sol";
+import "./Constants.sol";
+
+/**
+ * @title Proxy to allow voting from another address.
+ * @dev Allows a UMA token holder to designate another address to vote on their behalf.
+ * Each voter must deploy their own instance of this contract.
+ */
+contract DesignatedVotingV2 is Stakeable {
```

Comment:
If this isn't multicallable, execute unstake always has to be a separate transaction from the withdraw. Do you think that's enough reason to make it multicall?

Reply:
good call! it should be marked as multicall. I will leave this for a follow on PR where we add it to the other UMA.20 contracts(`Voting.sol`).